content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -403,4 +403,8 @@ nodeRegistration:
 Alternatively, you can modify `/etc/fstab` to make the `/usr` mount writeable, but please
 be advised that this is modifying a design principle of the Linux distribution.
 
+## `kubeadm upgrade plan` prints out `context deadline exceeded` error message
 
+This error message is shown when upgrading a Kubernetes cluster with `kubeadmin`in the case of running an external etcd. This is not a critical bug. You can proceed with `kubeadm upgrade apply ...`. 
```

Comment:
please change:
>  cluster with `kubeadmin`

to:
> cluster with kubeadm

and:
> This is not a critical bug. 

to:
> This is not a critical bug and happens because older versions of kubeadm perform a version check on the external etcd cluster.

Reply:
Will do.